graphql-apollo-server-tutorial.md

```diff
@@ -2379,27 +2372,27 @@ const user = (sequelize, DataTypes) => {
-* read more about {{% a_blank "JSON web tokens (JWT)" "https://jwt.io/" %}}
-* login (sign in) a user with GraphQL Playground
+* Register (sign up) a new user with GraphQL Playground
+* Check your users and their hashed passwords in the database with `psql`
+* Read more about {{% a_blank "JSON web tokens (JWT)" "https://jwt.io/" %}}
+* Login (sign in) a user with GraphQL Playground
   * copy and paste the token to the interactive token decoding on the JWT website (conclusion: the information itself isn't secure, that's why you shouldn't put a password in the token)
 
 {{% chapter_header "Authorization with GraphQL and Apollo Server" "apollo-server-authorization" %}}
 
-In the last section, you have set up all GraphQL mutations to enable the authentication with the server. You can register a new user with bcrypt hashed passwords and you are able to login with your user's credentials. Both authentication related GraphQL mutations return a token (JWT) for you which secures non-sensible user information with a secret. So what can be done once you have obtained the token in your client application?
+In the last section, you set up GraphQL mutations to enable authentication with the server. You can register a new user with bcrypt hashed passwords and you can login with your user's credentials. Both GraphQL mutations related to authentication return a token (JWT) that secures non-sensitive user information with a secret.
 
-The token, whether obtained on registration or login, is returned to the client application after a successful GraphQL `signIn` or `signUp` mutation. The client application has to make sure to store the token somewhere (e.g. [the browser's session storage](https://www.robinwieruch.de/local-storage-react)). Then, every time a request is made to the GraphQL server, the token has to be attached to the HTTP header of the HTTP request. Then the GraphQL server can validate the HTTP header, verify its authenticity, and perform the actual request (e.g. GraphQL operation). If the token is not valid (anymore), the GraphQL server has to return an error for the GraphQL client. If the client still has a token locally stored, it should remove the token from the storage and redirect the user to the login page.
+The token, whether its obtained on registration or login, is returned to the client application after a successful GraphQL `signIn` or `signUp` mutation. The client application must store the token somewhere like [the browser's session storage](https://www.robinwieruch.de/local-storage-react)). Every time a request is made to the GraphQL server, the token has to be attached to the HTTP header of the HTTP request. The GraphQL server can then validate the HTTP header, verify its authenticity, and perform a request like a GraphQL operation. If the token is invalid, the GraphQL server must return an error for the GraphQL client. If the client still has a token locally stored, it should remove the token and redirect the user to the login page.
```

Comment:
(https://www.robinwieruch.de/local-storage-react)) <- one paranthesis at the end is too much